### ISSUE TITLE
Confusing Braille Output for unpitched Percussion on single line staff 

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32324]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

The Braille output for single line unpitched percussion does not conform to any Braille standard.  It is understandable as it currently uses a different pitch for each different instrument (it has been suggested these might be the closest GM assignment if you ignore sharp/flat).  This does make it seem unnecessarily complicated to read.  Most transcribers in most countries simply notate all of these on C without any octave signs, but if MuseScore wanted to make them all A, the same as the score input note that would be the second best option.

STEPS:
1.  Create a score with a variety of single 1 line staff unpitched percussion instruments such as tambourine, claves, triangle, bass drum.
2. Enter a few notes on each and notice on the Braille panel they show as follows:
Triangle A5
Claves E5
Tambourine F3
Bass drum C2

The bass drum does make some sense as the most common Braille assignment for drum kit is to give each the equivalent letter of the staff if they were being read in Bass clef.  although manuals say the transcriber can choose a pitch of their choice, this seems to be the most common practice.  It is essential that changes to single line unpitched percussion to create them on just 1 note are not also applied to drum kit or other combos with multiple instruments on the one staff, they do need to continue to output with the varied pitches.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
See also 32323 No input of unpitched percussion from Braille panel using 6-key entry.
